server/routes/mention.js

```diff
@@ -21,10 +25,16 @@ router.get("/:id", async (req, res) => {
   }
 });
 
-router.get("/:companyName", async (req, res) => {
+router.get("/company/:companyName", async (req, res) => {
   try {
     const { companyName } = req.params;
-    const result = await Mention.find({ companyName });
+    const contentResult = await Mention.find({
+      content: new RegExp(companyName, "i"),
+    });
+    const titleResult = await Mention.find({
+      title: new RegExp(companyName, "i"),
+    });
+    const result = contentResult.concat(titleResult);
```

Comment:
probably want to combine the two regExp rather than doing one by one than concat because it is faster

Reply:
Good catch, thanks! I'll give that a shot now.

Reply:
Doing:

```javascript
const result = await Mention.find({
      title: new RegExp(companyName, "i"),
      content: new RegExp(companyName, "i"),
    });
   //...
});
```

...didn't work. It only returns results where the companyName is both in the title AND in the content. But not if it's just in the title OR just in the content.